app/guid-node/forks/template.hbs

```diff
@@ -0,0 +1,47 @@
+{{title (t 'forks.page_title' nodeTitle=model.taskInstance.value.title)}}
+<div class="container Forks">
+    {{#bs-modal open=newModal onSubmit=(action 'newFork') as |m|}}
+        {{#m.body}}
+            <h3>{{t 'forks.create_fork_modal'}}</h3>
+        {{/m.body}}
+        {{#m.footer}}
+            {{#bs-button onClick=(action (mut newModal) false)}}{{t 'general.cancel'}}{{/bs-button}}
+            {{#bs-button onClick=(action m.submit) type='info'}}{{t 'forks.fork'}}{{/bs-button}}
+        {{/m.footer}}
+    {{/bs-modal}}
+    {{delete-node-modal delete=(action 'delete') openModal=deleteModal closeModal=(action 'closeDeleteModal') nodeType=nodeType}}
+    <div class="Forks__header page-header visible-xs">
+        <h2 class="text-300">{{t 'forks.title'}}</h2>
+    </div>
+    <div class="Forks__nav m-md">
+        <a href="/{{model.taskInstance.value.id}}/analytics" class="padded strong">
+            {{fa-icon 'arrow-left'}} {{t 'forks.back'}}
+        </a>
+    </div>
+    <div class="Forks__forks">
+        {{#if getForks.isRunning}}
+            {{loading-indicator dark=true}}
+        {{else}}
+            {{#if forks.length}}
+                <ul class="list-group m-md ">
+                    <span id="forks" class="render-nodes-list">
+                        {{#each forks as | fork |}}
+                            <li class="project list-group-item list-group-item-node cite-container">
+                                {{node-blurb node=fork blurbType='fork' delete=(action openDeleteModal fork)}}
+                            </li>
+                        {{/each}}
+                    </span>
+                </ul>
+                {{simple-paginator maxPage=maxPage nextPage=(action 'next') previousPage=(action 'previous') curPage=page}}
+            {{else}}
+                {{t 'forks.no_forks'}}
+            {{/if}}
+        {{/if}}
+
+    </div>
+    <div class="Forks__new-fork">
+        <div class="m-md">
+            {{#bs-button type='success' onClick=(action (mut newModal) true)}}{{t 'forks.new'}}{{/bs-button}}
```

Comment:
You can click "New Fork" repeatedly while the first new fork is processing. That's handy for testing pagination but it should be disabled, ideally with some indication that it's in process. Maybe even a loading icon inside the button?

Reply:
Unfortunately we don't know when the new fork creation is over. Unless I'm misunderstanding you, we have to allow multiple forks creation to happen simultaneously

Reply:
No, I just mean until the request comes back. The toast notification doesn't show up until we get a response, and you can create several forks before there's any visual feedback that something has happened.

Reply:
(probably more of an issue locally than elsewhere, because slow)

Reply:
Yeah, new fork creation is a "Check back later to find out the status of your fork" sort of deal. It should definitely be disabled while the request is waiting to return, and there should be a message saying to check back later if there isn't.

Reply:
ah gotcha. yep, can do. Maybe disabling the button until either an error or success response?

Reply:
Yeah, and that should be true for pretty much every button ever. hashtag noExaggeration

Reply:
Yeah, disable it, and I think it'd be good to use [e-bootstrap's `loadingText`](https://www.ember-bootstrap.com/api/classes/Components.Button.html) property to give clearer feedback about what's happening (for pretty much every button ever). Maybe even add a [spinny icon](https://codepen.io/jmalatia/pen/HkmaA)? But that's a broader decision that should be consistent across the app.